server/utils/crmFieldFormatter.test.ts

```diff
@@ -24,6 +24,11 @@ describe('CRM Field Formatter', () => {
       expect(result).toEqual('14 November 2024')
     })
 
+    it('should format date in dd-mm-yyyy format', () => {
+      const result = formatDate('2024-11-14T00:00:00.000+00:00', formatDate('dd-MM-yyyy'))
```

Comment:
formatDate is being called twice here, it just needs to be:-
```suggestion
      const result = formatDate('2024-11-14T00:00:00.000+00:00', 'dd-MM-yyyy')
```